src/runtime/tests/type-test.ts

```diff
@@ -430,6 +431,173 @@ describe('types', () => {
       assert.notExists(a.canReadSubset);
       assert.strictEqual(a.resolution, sup);
     });
+    it(`successfully restricts full type ranges`, () => {
+      const varInfo1 = new TypeVariableInfo('x',
+          new EntityType(new Schema(['Foo'], {a: 'Text'})),
```

Comment:
I think the convention in this file is to use `EntityType.make` instead of directly using constructors. 
Whether this is a pattern we should continue, I'm not sure.

Reply:
using EntityType `ctor` directly is ok too (https://github.com/PolymerLabs/arcs/search?q=%22new+EntityType%28%22&unscoped_q=%22new+EntityType%28%22) , the difference is that EntityType.make constructs a Schema object from manifest artifacts, where `ctor` takes a Schema object.

Reply:
Cool, I'm happy with that.